manager/dm-server/src/main/java/org/apache/doris/stack/control/manager/ResourceNodeAndAgentManager.java

```diff
@@ -465,12 +462,7 @@ private boolean checkEventBeCancelled(long eventId) {
      * get server address
      */
     private String getServerAddr() {
-        String host = null;
-        try {
-            host = InetAddress.getLocalHost().getHostAddress();
-        } catch (UnknownHostException e) {
-            throw new ServerException("get server ip fail");
-        }
+        String host = System.getenv(EnvironmentDefine.STUDIO_IP_ENV);
```

Comment:
It is recommended that STUDIO_IP is optional, keep the previous logic, and replace when STUDIO_IP is not empty